### PR TITLE
Add ontology schema examples and user-friendly terms.

### PR DESCRIPTION
**Pull request by malloryfreeberg**
### Release notes
For all ontology schemas:
- add 2 examples
- add user-friendly term

### Reviews
- Needs 1 Reviewer
- Reviewer to merge changes

### CODE REVIEW

File: json_schema/module/ontology/development_stage_ontology.json [hunk 1]
```diff
@@ -31,12 +34,15 @@
                 "relations": ["rdfs:subClassOf"],
                 "direct": false,
                 "include_self": false
-            }
+            },
+            "user_friendly": "Development stage ontology ID",
+            "example": "EFO:0001272; EFO:0008562"
```
Comment (daniwelter) [inline json_schema/module/ontology/development_stage_ontology.json]: Change to human adult stage/HsapDv:0000087 and Theiler stage 28/EFO:0002588

File: json_schema/module/ontology/length_unit_ontology.json [hunk 2]
```diff
@@ -33,11 +35,15 @@
                 "relations": ["rdfs:subClassOf"],
                 "direct": false,
                 "include_self": false
-            }
+            },
+            "user_friendly": "Length unit ontology ID",
+            "example": "UO:0000017; EFO:0004387"
```
Comment (daniwelter) [inline json_schema/module/ontology/length_unit_ontology.json]: change inch to meter/UO:0000008

File: json_schema/module/ontology/protocol_type_ontology.json [hunk 3]
```diff
@@ -32,12 +35,14 @@
...
                 "include_self": false
-            }
+            },"example": "EFO:0009088; EFO:0009089",
+            "user_friendly": "Protocol type ontology ID"
         },
         "ontology_label": {
             "description": "The preferred label for the ontology term referred to in the ontology field. This may differ from the user-supplied value in the text field",
-            "type": "string"
+            "type": "string",
+            "example": "dissociation; enrichment",
```
Comment (daniwelter) [inline json_schema/module/ontology/protocol_type_ontology.json]: correct labels to dissociation protocol and enrichment protocol

Comment (zperova) [reply json_schema/module/ontology/protocol_type_ontology.json]: that's my bad, I can fix it now @malloryfreeberg

Comment (zperova) [reply json_schema/module/ontology/protocol_type_ontology.json]: all done now @malloryfreeberg

**Pull request merged**